### DISCUSSION TITLE
Unit Testing .NET Core with XUnit - Part One | Dan Does Code

### DISCUSSION THREAD

**Discussion by giscus-migrator** [danielmackay/nextjs-personal-blog#61] [Blog Comments]
Unit Testing .NET Core with XUnit - Part One
        
[http://www.dandoescode.com/blog/unit-testing-dotnet-core-with-xunit-part-one/](http://www.dandoescode.com/blog/unit-testing-dotnet-core-with-xunit-part-one/)

**Comment by SamBWagner** [comment]
A really good, succinct explanation of not only xunit but also Unit tests in general. 🙌

**Reply by danielmackay** [reply]
@SamBWagner thanks for the feedback.  Glad you found it useful. 😄
